### PR TITLE
Update view-reports-for-atp.md

### PR DESCRIPTION
**Pull request by faoquong**
@chrisda

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/security/office-365-security/view-reports-for-atp.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ These views give you the option to export, via a button click (in **Email** \> *
 
 ![This graphic shows Export as an option in the menu for the Malware view, right between Create Schedule, and Request Report.](../../media/tp-threatProtectStatRpt-BreakDownByExport.png)
 
-**Note**: The maximum number of entries that can be exported for **Phish** and **Malware** is just under 10000. If you export a view, only the most recent 10000 entries are exported.
+**Note**: The maximum number of entries that can be exported for **Phish** and **Malware** is just under 10000. If you export a view, only the most recent 10000 entries are exported. On the exported data, the 'message count' column represent the count of messages that hit the detection technology and policy type.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/view-reports-for-atp.md]:
```suggestion
> [!NOTE]
> The maximum number of entries that can be exported for **Phish** and **Malware** is just under 10000. If you export a view, only the most recent 10000 entries are exported. On the exported data, the _message count_ column represents the number of messages detected by the detection technology and policy type.    
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by mypil** [comment]
@faoquong  - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**